src/scripts/ccget

```diff
@@ -1,14 +1,9 @@
-#!/usr/bin/env python3
```

Comment:
As far as I know, this is the best approach. Python 3 might not be installed, and 2/3 doesn't matter so much here since the scripts aren't meant to be user-modified anyways. It's probably a coincidence that there's never been a problem with this before.